ui/src/pages/ShoeMap.tsx

```diff
@@ -108,10 +110,13 @@ export const ShoeMap: React.FC = () => {
             openDrawer={() => setIsDrawerOpen(StoryDrawerState.Open)}
           />
           <ZoomControl position="bottomright" />
-          <AttributionControl position="bottomleft" />
+          <AttributionControl position="topleft" />
```

Comment:
Maybe this should be topright since the filter will be at the topleft? Otherwise looks good.

Reply:
o true nice catch